### ISSUE TITLE
[en] better message for A_RB_NN

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#2546]
> The minerals present suggest derivation from a predominantly granite source.

<img width="410" alt="Bildschirmfoto 2020-03-04 um 09 38 59" src="https://user-images.githubusercontent.com/37363/75860446-24470f80-5dfc-11ea-97eb-7341430933b8.png">

The message is, at least for me, hard to read/understand.

**Comment by danielnaber** [comment]
Feedback from Alistair: "from a source that is predominantly granite" would be better. I'm working on < `filter` that suggests the adjective, but that won't help here.

**Comment by tiff** [comment]
Yes. I was referrring to the description which is hard to understand
